packages/core/src/components/drawer/drawer.tsx

```diff
@@ -67,20 +67,19 @@ export class Drawer {
    */
   @Event() drawerClose!: EventEmitter;
 
-  toggle = false;
-
   private static duration = 300;
   private callback = this.clickedOutside.bind(this);
   private divElement?: HTMLElement;
 
   @State() showContent = true;
+  // Added for unified state control
+  @State() internalVisible = false;
```

Comment:
I would prefer to use the existing `toggle` variable and add there the `@State` decorator to it.